system/service/role.go

```diff
@@ -604,6 +611,10 @@ func (svc role) Unarchive(ctx context.Context, roleID uint64) (err error) {
 	return svc.recordAction(ctx, raProps, RoleActionUnarchive, err)
 }
 
+func (svc role) CloneRules(ctx context.Context, roleID uint64, cloneToRoleID ...uint64) (err error) {
+	return svc.rbac.CloneRulesByRoleID(ctx, roleID, cloneToRoleID...)
+}
```

Comment:
Do we not need any access control here?
I'd imagine we should check for the can grant perm